docs/index.md

```diff
@@ -11,36 +11,69 @@ layout: default
 </div>
 </header>
 
-<section class="grey-bg">
+<section>
 <div class="wrapper">
-<div class="responsive-row center">
-<div style="max-width: 600px">
+<h1 class="title">
+About
+</h1>
 
-## Fast, lightweight web components
+<div class="responsive-row">
 
-LitElement is a simple base class for creating fast, lightweight web components that work in any web page with any framework.
+<div style="flex:1;">
+<h2>
+Fast, lightweight web components
+</h2>
+<p>
+LitElement is a simple base class for creating fast and lightweight web components that work in any web page with any framework.
+</p>
+</div>
 
-For rendering, LitElement uses [lit-html](https://lit-html.polymer-project.org/)–a fast HTML templating library. To build an app out of LitElement components, check out [PWA Starter Kit](https://pwa-starter-kit.polymer-project.org/).
+<div style="flex:1;">
+<h2>
+Using lit-html
+</h2>
+<p>
+For rendering, LitElement uses [lit-html](https://lit-html.polymer-project.org/) – a fast HTML templating library. To build an app out of LitElement components, check out [PWA Starter Kit](https://pwa-starter-kit.polymer-project.org/).
```

Comment:
Google style guide says no spaces around the dash https://developers.google.com/style/dashes

Looks like I need to make it an em dash (—) instead of an en dash (-) tho